app/models/account/service_discovery_support.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Account::ServiceDiscoverySupport
```

Comment:
Please check https://github.com/3scale/porta/blob/master/app/models/service_discovery/model_extensions.rb. Either this or that should be refactored. It can be confusing otherwise.

Reply:
This has been put in `app/lib/service_discovery`

You are right about refactor. But I'd like some advice about, putting it in `app/lib/service_discovery` like part of a whole feature or extending models and then putting it `app/models/account/` 
What is the more readable to you?